cpp/ql/src/semmle/code/cpp/models/interfaces/FlowSource.qll

```diff
@@ -11,11 +11,21 @@ import FunctionInputsAndOutputs
 import semmle.code.cpp.models.Models
 
 /**
- * A library function which returns data read from a network connection.
+ * A library function which returns data that may be read from a network connection.
```

Comment:
```suggestion
 * A library function that returns data that may be read from a network connection.
```

A further rearrangement of the sentence could perhaps avoid having two occurrences of "that", but grammatically it can't be "which" here. [[1]](https://github.com/github/codeql/blob/master/docs/qldoc-style-guide.md#qldoc-for-classes) [[2]](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/that-vs-which).